masterlist.yaml

```diff
@@ -14007,6 +14007,11 @@ plugins:
       - crc: 0x12897011
         util: 'SSEEdit v4.0.0'
 
+  - name: 'Honorhall Fixes and Expansion.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/66955/' ]
+    after: [ 'HearthfireMultiKid.esp' ]
```

Comment:
I'm not sure if this after is required tbh, LOOT already sorts them correctly.
Though you may have come across a case where it doesn't (Depends on the mods and rules you use).
So I'm not against it being added.

Oh and files should be placed in alphabetical order within sections.

```
  - name: 'Honorhall Fixes and Expansion.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/66955/' ]
    after: [ 'HearthfireMultiKid.esp' ]
```